.github/workflows/linux_ci.yml

```diff
@@ -377,7 +377,18 @@ jobs:
         run: |
           pip install .[test]
 
+      - name: Files related to Pinecone
+        id: pinecone-files
+        uses: tj-actions/changed-files@v18.4
+        with:
+          files: |
+            **/document_stores/pinecone.py
```

Comment:
Let's also add `**/document_stores/filter_utils.py` because of the usage of `LogicalFilterClause`

Reply:
Makes totally sense, added it :)